app/code/Magento/InventoryCatalog/Test/Integration/UpdateLegacyCatalogInventoryPluginTest.php

```diff
@@ -21,6 +22,9 @@
 use Magento\Indexer\Model\Indexer;
 use Magento\Inventory\Indexer\SourceItem\SourceItemIndexer;
 
+/**
+ * Class UpdateLegacyCatalogInventoryPluginTest
```

Comment:
It is more useful to provide doctype about functionality but not about class name